src/app/shared/forms/planet-markdown-textbox.scss

```diff
@@ -26,8 +26,8 @@ planet-markdown-textbox {
   }
 
   .CodeMirror.CodeMirror-fullscreen {
-    margin-top: 10px;
-    margin-left: 70px;
+    padding-top: 30px;
+    padding-left: 80px;
```

Comment:
padding left seems to be big. 10 px looks ok.

Reply:
You are right.  Probably I would need to add some media queries for that.